### ISSUE TITLE
Update manifest when changing file name in assets

### ISSUE THREAD

**Issue by garbinmarcelo** [KABBOUCHI/laravel-mix-merge-manifest#6]
Hi,

For example, I have a file named: style_file.css, when I compile webpack it is written to mix-manifest.json the path to which the file was compiled, right?

But if I rename this file to file-style.scss and try to compile again, it ends up giving error with webpack. Not sure if this package is the problem, but when I delete the lines of mix-manifest.json with the old path and recompile the assets, everything runs normally.

Would there be any way to solve this or is it a bug?

Thanks for the attention and the package, I started using it a short time ago.

Edit: Another question, is it possible to choose which files will be part of mix-manifest.json?

**Comment by KABBOUCHI** [comment]
Yep, its the only way to fix it,  just delete the `mix-manifest.json` when if u got any weird error from mix.

> Edit: Another question, is it possible to choose which files will be part of mix-manifest.json?

Sure, I will add it in few days or if you want sumbit a PR.
